src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -277,68 +260,72 @@ protected override Delegate MakeGetter(Row input, int iinfo, Func<int, bool> act
-            string weightColumn = null) : base(env, new PlattCalibratorTrainer(env), predictor, labelColumn, featureColumn, weightColumn)
+            string scoreColumn = DefaultColumnNames.Score,
+            string weightColumn = null) : base(env, new PlattCalibratorTrainer(env), labelColumn, scoreColumn, weightColumn)
         {
-
         }
 
-        protected override CalibratorTransformer<PlattCalibrator> Create(IHostEnvironment env, PlattCalibrator calibrator)
-        => new PlattCalibratorTransformer(env, calibrator);
+        [BestFriend]
+        private protected override CalibratorTransformer<PlattCalibrator> Create(IHostEnvironment env, PlattCalibrator calibrator)
+            => new PlattCalibratorTransformer(env, calibrator);
     }
 
     /// <summary>
-    /// Obtains the probability values by fitting the sigmoid:  f(x) = 1 / (1 + exp(-slope * x + offset).
+    /// Obtains the probability values by applying the sigmoid:  f(x) = 1 / (1 + exp(-slope * x + offset).
+    /// Note that unlike, say, <see cref="PlattCalibratorEstimator"/>, the fit function here is trivial
+    /// and just "fits" a calibrator with the provided parameters.
     /// </summary>
     /// <remarks>
-    /// For the usage pattern see the example in <see cref="CalibratorEstimatorBase{TCalibratorTrainer, TICalibrator}"/>.
+    /// For the usage pattern see the example in <see cref="CalibratorEstimatorBase{TICalibrator}"/>.
     /// </remarks>
-    public sealed class FixedPlattCalibratorEstimator : CalibratorEstimatorBase<FixedPlattCalibratorTrainer, PlattCalibrator>
+    public sealed class FixedPlattCalibratorEstimator : CalibratorEstimatorBase<PlattCalibrator>
     {
         /// <summary>
-        /// Initializes a new instance of <see cref="FixedPlattCalibratorEstimator"/>
+        /// Initializes a new instance of <see cref="FixedPlattCalibratorEstimator"/>.
         /// </summary>
+        /// <remarks>
+        /// Note that unlike many other calibrator estimators this one has the parameters pre-specified.
+        /// This means that it does not have a label or weight column specified as an input during training.
+        /// </remarks>
         /// <param name="env">The environment to use.</param>
-        /// <param name="predictor">The predictor used to train the data.</param>
         /// <param name="slope">The slope in the function of the exponent of the sigmoid.</param>
         /// <param name="offset">The offset in the function of the exponent of the sigmoid.</param>
-        /// <param name="labelColumn">The label column name.</param>
-        /// <param name="featureColumn">The feature column name.</param>
-        /// <param name="weightColumn">The weight column name.</param>
+        /// <param name="scoreColumn">The score column name. This is consumed both when this estimator
+        /// is fit and when the estimator is consumed.</param>
         public FixedPlattCalibratorEstimator(IHostEnvironment env,
```

Comment:
>public [](start = 8, length = 6)

are we leaving the ctors of the calibratorEstimators public?

Reply:
No. However, until #1871 is completed with the proper catalog for calibrators added, this should remain public. Incidentally this was what I meant by this sentence at the end of my PR description:

> Note that ultimately the trainer estimator constructors _will_ be internal, pending #1871.